app/controllers/openid_connect/authorization_controller.rb

```diff
@@ -210,13 +211,13 @@ def track_events
         service_provider: @authorize_form.service_provider,
         user: current_user,
       )
-
       analytics.sp_redirect_initiated(
         ial: event_ial_context.ial,
         billed_ial: event_ial_context.bill_for_ial_1_or_2,
         sign_in_flow: session[:sign_in_flow],
         vtr: sp_session[:vtr],
         acr_values: sp_session[:acr_values],
+        sign_in_duration_seconds: sign_in_duration,
```

Comment:
Maybe we should call the method `sign_in_duration_seconds` to (1) align with the property we assign it to, (2) to make it clearer what value is returned by that method, and (3) to be able to use the hash shorthand?

```suggestion
        sign_in_duration_seconds:,
```

Reply:
I like that much better. Thanks! 👍